resources/images/gun_data.py

```diff
@@ -1,52 +0,0 @@
-"""
-@file gun_data.py
```

Comment:
Hm I added a method in my most recent PR, prepare for uncomfortable merge issues

Reply:
Yee I'll work it out.

Reply:
Actually should be fine. I shifted over logic to a GunImage class under classes/, so can accept your changes.
Then I'll merge them together for consistency.